### PR TITLE
Add load/save to eeprom functionality for tuning parameters.

### PR DESCRIPTION
**Pull request by ranechita**
Signed-off-by: Ramona Alexandra Nechita <ramona.nechita@analog.com>

The tuning parameters from the calibration weren't save anywhere previously, therefore two utility functions for loading and saving to eeprom were added.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [ ] 

**Test Configuration**:
* Hardware: ADRV9361-ZC702
* OS: Ubuntu 20.04

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have signed off all commits and they contain "Signed-off by: <insert name>"
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: test/scpi.py [hunk 1]
```diff
@@ -55,7 +56,7 @@ def select_channel(instr):
 
 def get_freq(instr):
     # set gate time to 600 for better precision
-    instr.write("SMT600")
+    # instr.write("SMT600")
```
Comment (tfcollins) [inline test/scpi.py:59]: Remove

File: test/scpi.py [hunk 2]
```diff
@@ -93,14 +94,89 @@ def get_clk_rate(classname, iio_uri):
     return float(clk_rate)
 
 
+def save_to_eeprom(iio_uri, coarse, fine):
+    full_uri = iio_uri.split(":", 2)
+    if full_uri[0] != "ip":
+        print("Tuning currently supported only for ip URIs")
```
Comment (tfcollins) [inline test/scpi.py:100]: Should this generate an exception?

File: test/scpi.py [hunk 3]
```diff
@@ -93,14 +94,89 @@ def get_clk_rate(classname, iio_uri):
...
+    if ssherr.channel.recv_exit_status() != 0:
+        raise paramiko.SSHException("fru-dump command did not execute properly")
+
+    ssh_client.close()
+
+
+def load_from_eeprom(iio_uri):
+    full_uri = iio_uri.split(":", 2)
+    if full_uri[0] != "ip":
+        print("Tuning currently supported only for ip URIs")
```
Comment (tfcollins) [inline test/scpi.py:132]: Same as comment above

File: test/scpi.py [hunk 4]
```diff
@@ -133,6 +209,7 @@ def dcxo_calibrate(classname, iio_uri):
                     step = int(round(-1 * ((step * diff) / abs(diff - prev_diff)) / 2))
                 coarse += step
             else:
+                # coarse += 1
```
Comment (tfcollins) [inline test/scpi.py:212]: Remove

File: test/scpi.py (hunk 2 above)
Comment (ranechita) [reply test/scpi.py:100]: I don't know if it should generate an exception necessarily, maybe a pytest.skip is more appropriate?

Comment (tfcollins) [reply test/scpi.py:99]: sure that works too

**Pull request merged**